### PR TITLE
Add SEO Optimization

### PR DESCRIPTION
**Pull request by ishaanbedi**
## Related Issue

- Integrated next-seo with the project.

Closes: #202 

### Describe the changes you've made

Installed next-seo to the project, created a `next-seo.config.js` file to globally add meta tags and og tags, and manually added relevant next-seo configuration in all the pages.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## How Has This Been Tested?

Verified for each page by checking the source of the file, both in IDE & browser.

## Checklist
- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

<a href="https://gitpod.io/#https://github.com/WebXDAO/WebXDAO.github.io/pull/204"></a>

### CODE REVIEW

File: src/pages/_app.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
-import WebXLayout from '../layouts/WebXLayout';
-import '../styles/globals.css';
-
+import WebXLayout from "../layouts/WebXLayout";
+import "../styles/globals.css";
+import SEO from "../../next-seo.config";
+import { DefaultSeo } from "next-seo";
 function MyApp({ Component, pageProps }) {
```
Comment (krshkodes) [inline src/pages/_app.js:5]: Let's add a empty line here, nothing but just a style preference

```suggestion
import { DefaultSeo } from "next-seo";

function MyApp({ Component, pageProps }) {
```

File: src/pages/blog.js [hunk 2]
```diff
@@ -1,12 +1,25 @@
-import Head from 'next/head';
+import Head from "next/head";
 // import { Blogs } from '../components/Global'
-
+import { NextSeo } from "next-seo";
 export default function Blog({ articles }) {
+  const SEO = {
```
Comment (krshkodes) [inline src/pages/blog.js:5]:
```suggestion
import { NextSeo } from "next-seo";

export default function Blog({ articles }) {
  const SEO = {
```

File: src/pages/blog.js [hunk 3]
```diff
@@ -21,8 +34,8 @@ export default function Blog({ articles }) {
 }
 
 export async function getStaticProps() {
-  const res = await fetch('https://dev.to/api/articles?username=webxdao', {
-    method: 'GET',
+  const res = await fetch("https://dev.to/api/articles?username=webxdao", {
+    method: "GET",
```
Comment (krshkodes) [inline src/pages/blog.js:38]:
```suggestion
  const res = await fetch('https://dev.to/api/articles?username=webxdao', {
    method: 'GET',
```

**Review by krshkodes — COMMENTED** [review]
Please add the lines between the imports and the function + remove all the formatting based changes made like double quotes instead of single quotes

**Review by krshkodes — APPROVED** [review]
I think the code-style issues can be fixed in another PR 👍

**Comment by krshkodes** [comment]
Merging to the SEO branch because more sub-tasks need to completed

**Comment by ishaanbedi** [comment]
> WebXDAO:seo

Great! Looking forward to contributing more!

**Comment by ishaanbedi** [comment]
> Merging to the SEO branch because more sub-tasks need to completed

Great! Looking forward to contribute more.

**Review by vinzvinci — COMMENTED** [review]
Hey, this looks good. Sorry I'm late. Thanks for your contributions!

**Pull request merged**